### PR TITLE
Add/rename scripts to build browser and node targets

### PR DESCRIPTION
**Pull request by tiran**

### CODE REVIEW

**Review by brettcannon — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,39 +37,69 @@ jobs:
...
+            cpython/builddir/emscripten-browser/pyconfig.h
+            cpython/builddir/emscripten-browser/libpython*.a
+            cpython/builddir/emscripten-browser/Modules/Setup.local
+            cpython/builddir/emscripten-browser/Modules/Setup.stdlib
+            cpython/builddir/emscripten-browser/Modules/config.c
+            cpython/builddir/emscripten-browser/Modules/_decimal/libmpdec/libmpdec.a
+            cpython/builddir/emscripten-browser/Modules/expat/libexpat.a
+            cpython/builddir/emscripten-browser/Programs/python.o
+          if-no-files-found: error
+      - name: "Upload wasm artifacts"
```
Comment (emmatyping) [inline .github/workflows/ci.yml]:
```suggestion
      - name: "Upload node build artifacts"
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -37,39 +37,69 @@ jobs:
...
         run: |
-          ls -la --si cpython/builddir/host/python*
-          ls -la cpython/builddir/host/Modules/
-          test -e cpython/builddir/host/python.data || exit 1
+          ls -la --si cpython/builddir/emscripten-browser/python*
+          ls -la --si cpython/builddir/emscripten-node/python*
+          ls -la cpython/builddir/emscripten-browser/Modules/
+          test -e cpython/builddir/emscripten-browser/python.data || exit 1
+          test -e cpython/builddir/emscripten-node/python.wasm || exit 1
       - name: "Upload wasm artifacts"
```
Comment (emmatyping) [inline .github/workflows/ci.yml]:
```suggestion
      - name: "Upload browser build artifacts"
```

File: build-python-emscripten-browser.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/env bash
+
+shopt -s extglob
```
Comment (emmatyping) [inline build-python-emscripten-browser.sh]:
```suggestion
```

We don't need this anymore :)

File: build-python-emscripten-node.sh [hunk 4]
```diff
@@ -2,18 +2,19 @@
 
 shopt -s extglob
```
Comment (emmatyping) [inline build-python-emscripten-node.sh]:
```suggestion
```

**Review by emmatyping — APPROVED** [review]
Looks great! Thanks for fixing these :)

Just three small suggestions to make.

**Pull request merged**